src/Mod/Part/App/TopoShape.h

```diff
@@ -629,6 +655,58 @@ class PartExport TopoShape: public Data::ComplexGeoData
      */
     TopoShape &makeElementCompound(const std::vector<TopoShape> &shapes, const char *op=nullptr, bool force=true);
 
+    TopoShape &makeElementFace(const std::vector<TopoShape> &shapes,
+                        const char *op = nullptr,
+                        const char *maker = nullptr,
+                        const gp_Pln *pln = nullptr);

```

Comment:
Needs formatting cleanup -- does your IDE allow selective application of clang-format? I don't want to do the whole file, just all the new code.